### PR TITLE
Bug fix on transaction activity 

### PR DESCRIPTION
**Pull request by Jibz1**
Fix blank txn activity feed

### CODE REVIEW

File: apps/wallet/src/ui/app/components/receipt-card/index.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ function ReceiptCard({ txDigest }: TxResponseProps) {
                         {assetCard}
                     </div>
 
-                    {transferMeta[transferType].address && (
+                    {transferMeta[transferType].address ? (
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/components/receipt-card/index.tsx:164]: Why is this change needed?

**Review by Jordan-Mysten — APPROVED** [review]
One question about a change that doesn't seem needed but looks good otherwise

File: apps/wallet/src/ui/app/components/receipt-card/index.tsx (hunk 1 above)
Comment (Jibz1) [reply apps/wallet/src/ui/app/components/receipt-card/index.tsx:164]: Oh, remove extra 0. 
Screenshot 2023-01-14 at 10 23 41 AM
